journals/apps/journals/models.py

```diff
@@ -314,6 +315,17 @@ def generate_basket_url(self, sku):
         ))
         return basket_url
 
+    def update_related_objects(self, deactivate=False):
+        data = {
+            "status": "active" if not deactivate else "inactive",
+            "card_image_url": self.site.root_url + self.card_image.file.url,
```

Comment:
just got an error when running the publish_journals mgmt command:
```
 File "/edx/app/journals/journals/journals/apps/journals/models.py", line 327, in update_related_objects
    "card_image_url": self.site.root_url + self.card_image.file.url,
AttributeError: 'NoneType' object has no attribute 'file'
```
We could check for None but might be better if we set a default card image on the about page as a default value. That way something will show up in the card on the index page and in LMS card image for the Journal